SUMMARY.md

```diff
@@ -122,14 +122,21 @@
   - [Deploying C2D](infrastructure/compute-to-data-minikube.md)
   - [C2D - Private Docker Registry](infrastructure/compute-to-data-docker-registry.md)
 - [🤑 DeFi](defi/README.md)
-- [💰 Data Farming Rewards](rewards/README.md)
-  - [Farming 101](rewards/df-intro.md)
-  - [veOCEAN](rewards/df-veocean.md)
-  - [Curate with Volume DF](rewards/df-volumedf.md)
-  - [Predict with Challenge DF](rewards/df-challengedf.md)
-  - [Complete User Guides](rewards/df-guides.md)
-  - [Emissions & APYs](rewards/df-emissions-apys.md)
-- [🤝 Contribute](contribute/README.md)
-  - [Partners & Collaborators](contribute/projects-using-ocean.md)
-  - [Contributor Code of Conduct](contribute/code-of-conduct.md)
-  - [Legal Requirements](contribute/legal-reqs.md)
+* [👀 Predictoor](predictoor/README.md)
+  * [Learn about Predictoor](predictoor/pdr-intro.md)
```

Comment:
Rename "Learn about to Predictoor" to "Predictoor Introduction". 

Why: _all_ the Predictoor docs are to learn about Predictoor! So this page needs to be more precise. It's really an introduction. (And the md file is already named that way, yay:)

Reply:
Done